src/hooks/useTab.ts

```diff
@@ -0,0 +1,25 @@
+import history from 'services/history';
```

Comment:
I've extracted the logic in Results.js so that it can be reused: this hook checks in the url for a hash in the form : #view=myCurrentTabKey. This hash can be used to "remember" what tab should be displayed.

Reply:
we have one problem now with Apollo and our routing mismatch version.

services/history will have to go and replace with `react-router-dom` 

(not blocking for the moment, but if we can use the router-dom, it will remove future technical debt